packages/lib/services/spellChecker/SpellCheckerService.ts

```diff
@@ -60,17 +70,23 @@ export default class SpellCheckerService {
 	}
 
 	private applyStateToDriver() {
-		this.driver_.setLanguage(this.enabled ? this.language : '');
+		this.driver_.setLanguages(this.enabled ? this.languages : []);
 	}
 
 	public setLanguage(language: string) {
```

Comment:
If I understand correctly, this will add the language if it's not already set, and remove it if it is. Is that correct? Is that case please could you name the function "toggleLanguage" (and update the callers too)?